### PR TITLE
bump hail to 0.2.128

### PR DESCRIPTION
**Pull request by jklugherz**

### CODE REVIEW

File: hail_search/deploy/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM hailgenetics/hail:0.2.126
+FROM hailgenetics/hail:0.2.128
```
Comment (jklugherz) [inline hail_search/deploy/Dockerfile:1]: will I need to create a requirements file and include `google-cloud-storage` as a dependency?

Comment (hanars) [reply hail_search/deploy/Dockerfile:1]: I hope not? This docker image should never be interacting with cloud storage

File: hail_search/requirements-test.in [hunk 2]
```diff
@@ -1,2 +1,3 @@
 coverage<5.2
 pytest-aiohttp
+google-cloud-storage>=2.14.0
```
Comment (hanars) [inline hail_search/requirements-test.in:3]: what in the hail search tests relies on cloud storage? The tests were passing before this commit, and I don'tt think. its neccessary to add this dependency

**Review by hanars — CHANGES_REQUESTED** [review]

File: hail_search/requirements-test.in (hunk 2 above)
Comment (bpblanken) [reply hail_search/requirements-test.in:3]: Yeah, I don't think we need this dependency on the seqr side.  This dependency is our gencode download in the pipeline.

Comment (jklugherz) [reply hail_search/requirements-test.in:3]: oh I was literally looking at a unit test run from another repo, my bad I'll remove this

**Review by hanars — APPROVED** [review]

**Pull request merged**